2.4/run-mongod.sh

```diff
@@ -7,6 +7,8 @@ set -e
 MAX_ATTEMPTS=60
 SLEEP_TIME=1
 
+MONGODB_CONFIG_PATH="/opt/openshift/etc/mongodb.conf"
```

Comment:
@mfojtik imo that would only confuse some users. And if somebody would want to change the path of the config file, he would need to do it even in Dockerfile